### PR TITLE
EZP-29249: Changed texts in relation fields

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29249
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added and changed texts in relations fields.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/fieldType/edit/_ezobjectrelationlist.scss [hunk 1]
```diff
@@ -50,6 +50,11 @@
         &__hidden-column {
             display: none;
         }
+
+        &__helper {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/fieldType/edit/_ezobjectrelationlist.scss]: I would name it `__helper-text`

File: src/bundle/Resources/views/fieldtypes/edit/ezobjectrelation.html.twig [hunk 2]
```diff
@@ -6,5 +6,6 @@
     {% set limit = 1 %}
     {% set default_location = form.vars.default_location ? form.vars.default_location.id : null %}
     {% set allowed_content_type_identifiers = form.parent.vars.value.fieldDefinition.fieldSettings.selectionContentTypes %}
+    {% set helper = 'ezobjectrelation.helper'|trans|desc('Stores a relation to a content object.') %}
```
Comment (dew326) [inline src/bundle/Resources/views/fieldtypes/edit/ezobjectrelation.html.twig]: Please extract translations.

Comment (DominikaK) [inline src/bundle/Resources/views/fieldtypes/edit/ezobjectrelation.html.twig]:
```suggestion
    {% set helper = 'ezobjectrelation.helper'|trans|desc('Stores a Relation to a Content item.') %}
```

File: src/bundle/Resources/views/fieldtypes/edit/ezobjectrelationlist.html.twig [hunk 3]
```diff
@@ -6,5 +6,6 @@
     {% set limit = form.parent.vars.value.fieldDefinition.validatorConfiguration.RelationListValueValidator.selectionLimit %}
     {% set default_location = form.vars.default_location ? form.vars.default_location.id : null %}
     {% set allowed_content_type_identifiers = form.parent.vars.value.fieldDefinition.fieldSettings.selectionContentTypes %}
+    {% set helper = 'ezobjectrelationlist.helper'|trans|desc('Stores relations to other content objects.') %}
```
Comment (DominikaK) [inline src/bundle/Resources/views/fieldtypes/edit/ezobjectrelationlist.html.twig]:
```suggestion
    {% set helper = 'ezobjectrelationlist.helper'|trans|desc('Stores Relations to other Content items.') %}
```

File: src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig [hunk 4]
```diff
@@ -79,8 +80,10 @@
     </div>
     <div class="ez-relations__cta" {% if has_relations %}hidden="true"{% endif %}>
         <p class="ez-relations__cta-text">
-        {% if limit != 0 %}
-            {{ 'ezobjectrelationlist.cta.limit'|trans({'%limit%': limit})|desc('Set up multiple relations with up to %limit% content items') }}
+        {% if limit != 1 %}
+            {{ 'ezobjectrelationlist.cta.limit.multi'|trans|desc('Set up multiple relations with one or several content items') }}
```
Comment (DominikaK) [inline src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig]:
```suggestion
            {{ 'ezobjectrelationlist.cta.limit.multi'|trans|desc('Set up multiple Relations with one or several Content items') }}
```

File: src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig [hunk 5]
```diff
@@ -79,8 +80,10 @@
...
+        {% else %}
+            {{ 'ezobjectrelationlist.cta.limit.single'|trans|desc('Set up a relation with one content item') }}
```
Comment (DominikaK) [inline src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig]:
```suggestion
            {{ 'ezobjectrelationlist.cta.limit.single'|trans|desc('Set up a Relation with one Content item') }}
```

**Comment by lserwatka** [comment]
@GrabowskiM could you merge it up?

**Pull request merged**